### PR TITLE
refactor: deprecate constants files

### PR DESCRIPTION
**Pull request by 404skillz**

### CODE REVIEW

File: test/utils/constants.ts
Comment (0xGorilla) [inline test/utils/constants.ts:1]: Don't you think all the other variables are useful??

**Review by 0xng — APPROVED** [review]

File: test/utils/constants.ts
Comment (404skillz) [reply test/utils/constants.ts:1]: They are all in ethersprojects constants file 😋 : https://github.com/ethers-io/ethers.js/blob/master/packages/constants/src.ts/index.ts

Comment (404skillz) [reply test/utils/constants.ts:1]: I'll wait for your comment before merge @0xGorilla

Comment (0xGorilla) [reply test/utils/constants.ts:1]: Great!! TY

**Review by 0xGorilla — APPROVED** [review]

**Pull request merged**
